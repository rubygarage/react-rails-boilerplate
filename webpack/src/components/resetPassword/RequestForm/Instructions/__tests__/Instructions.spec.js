import React from 'react'
import { shallow } from 'enzyme'
import Instruction from 'components/resetPassword/RequestForm/Instructions'

describe('<Instruction />', () => {

  it('renders Instruction component', () => {
    
    const component = shallow(
      <Instruction />
    )

    expect(component).toMatchSnapshot()
  })
})
