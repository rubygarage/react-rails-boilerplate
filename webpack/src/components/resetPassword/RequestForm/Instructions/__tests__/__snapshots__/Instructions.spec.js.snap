// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Instruction /> renders Instruction component 1`] = `
<main
  className="container"
>
  <div
    className="jumbotron"
  >
    <h3>
       
      <FormattedMessage
        id="restore_password.header"
        values={Object {}}
      />
       
    </h3>
    <p
      className="lead"
    >
       
      <FormattedMessage
        id="restore_password.instructions"
        values={Object {}}
      />
       
    </p>
  </div>
</main>
`;
